src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
-// write code here
+const employeesArray = Array.from(document.querySelectorAll('li'));
+const parentList = document.querySelector('ul');
+
+/**
+ * Helper function for turning a salary string into a valid integer.
+ * @param {*} Node
+ * @returns Integer
+ */
+function getSalaryInt(item) {
+  const salaryString = item.dataset.salary.replace(/,/g, '').replace('$', '');
+
+  return parseInt(salaryString) || 0;
+}
+
+/**
+ * Function thaat sorts emploees nodel by salary in descending order
```

Comment:
There are typos in the comment: 'thaat' should be 'that', 'emploees' should be 'employees', and 'nodel' should be 'nodes'.